help/unsubscribe-from-a-stream.md

```diff
@@ -30,7 +30,9 @@ You can always unsubscribe from any stream in Zulip.
 
 {relative|stream|subscribed}
 
-1. Click the green checkmark to the left of a stream to unsubscribe from it.
+1. Click the **checkmark**
+   (<img src="/static/images/help/desktop-web-check-icon.svg" alt="checkmark" class="help-center-icon"/>)
+   icon to the left of a stream to unsubscribe from it.
```

Comment:
Why do we need new copies of these assets rather than just accessing the ones we use in production? The latter is nice because it's less likely to break if we make future changes.

Oh, I see, the icons are just inline in the template. Hrm, OK.